src/hooks/useExtendedVoteData.js

```diff
@@ -18,7 +18,7 @@ export default function useExtendedVoteData(vote) {
 
   const dandelionVotingContract = useContractReadOnly(
     dandelionVotingAddress,
-    dandelionVotingAbi.abi
+    dandelionVotingAbi
```

Comment:
does it work like this? i think that we have the abi key on the json

Reply:
Ah yes, i updated the json file to only include the abi part, the rest was not needed.